diagnostic.md

```diff
@@ -6,21 +6,23 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    router: Set up associations between URLs and views by designating routes
+    Route: define the url via '{path: url}' and designate the template that should
+    be rendered.
     ```
```

Comment:
Those are actually all router tasks. The route object specifies how to load model data via the model hook.